win32/Lib/winnt.py

```diff
@@ -180,15 +180,15 @@ def SORTIDFROMLCID(lcid):
 
 
 def UNREFERENCED_PARAMETER(P):
-    return
+    return P
```

Comment:
why this change?

Reply:
Bad merge, it should be as was in this PR: https://github.com/mhammond/pywin32/pull/2050/files#diff-929c99ca621bd62184b36b3c6192c02547a50e4bc9e45a1968a2565622f454d2

Reply:
yeah - I did mean to say that these should just be removed - that PR removed duplicates of this odd definition, can this PR just remove them completely?